src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -319,7 +325,7 @@ doLedgerEntry(Context const& context)
 
         key = ripple::keylet::line(*id1, *id2, currency).key;
     }
-    else if (request.contains(JS(ticket)))
+    if (request.contains(JS(ticket)))
```

Comment:
The final `else` is still there - need to remove (I could swear you did that before?).